core/variant/variant_utility.cpp

```diff
@@ -695,9 +704,9 @@ struct VariantUtilityFunctions {
 		return p_arr.hash();
 	}
 
-	static inline Variant instance_from_id(int64_t p_id) {
+	static inline Object *instance_from_id(int64_t p_id) {
```

Comment:
@reduz I guess it's fine to return `Object *` directly? That's what the GDScript method does, and it seems to compile fine.